packages/next/package.json

```
@@ -53,6 +53,6 @@
   },
   "license": "SIL OPEN FONT LICENSE",
   "peerDependencies": {
-    "next": "^13.2 || ^14"
+    "next": ">=13.2.0 <15"
```

Let's just allow anything newer than 13.2.0.
```suggestion
    "next": ">=13.2.0"
```